app/views/home/index.html.erb

```diff
@@ -0,0 +1,18 @@
+	<section id="cd-intro">
+		<div id="cd-intro-background"></div>
+		<div id="cd-intro-tagline">
+			<h1>Welcome to CS</h1>
+		</div> <!-- cd-intro-tagline -->
+	</section> <!-- #cd-intro -->
+
+	<main class="cd-content">
+		<div class="cd-container">
+			<h2>We are here for you</h2>
+			<p>There are many exciting ways to complete a CS curriculum</p>
+			<p>Please use this portal to create a schedule to help the department best plan appropiate class offerings for the following semester.</p>
+			<p></p>
```

Comment:
Empty `<p>` tag, please remove (use `<br>` if you're trying to make a line break for space)